aim/sdk/plotly_sequence.py

```diff
@@ -0,0 +1,38 @@
+from typing import Union, Tuple
+
+from aim.sdk.sequence import Sequence
+from aim.sdk.objects.plotly import Plotly
+
+
+class Plotlys(Sequence):
```

Comment:
Plotlys  -> Figures

Reply:
yeah seems more logical - i was struggling to come up with a name 🤣